Sources/DistributedActors/Cluster/ClusterControl.swift

```diff
@@ -155,4 +155,77 @@ public struct ClusterControl {
     public func down(member: Cluster.Member) {
         self.ref.tell(.command(.downCommandMember(member)))
     }
+
+    public func joined(node: UniqueNode, within: Duration) async throws {
+        try await self.awaitMemberStatus(is: .up, within: within, node: node)
+    }
```

Comment:
That's a nice one; past tense works well here since it is `joining -> up` so makes sense; It'll be a common API to use I suspect, good to have a special version for it.

Can we perhaps also return the `Member` for the joined node?

Reply:
PS: Good to always force passing a duration, agreed 👍